cmd/root.go

```diff
@@ -114,6 +119,10 @@ any client SSL certificates.`,
 		"Path to a service account key to use for authentication.")
 	cmd.PersistentFlags().BoolVarP(&c.conf.GcloudAuth, "gcloud-auth", "g", false,
 		"Use gcloud's user configuration to retrieve a token for authentication.")
+	cmd.PersistentFlags().StringVarP(&c.prometheusNamespace, "prometheus-namespace", "n", "",
+		"Enable Prometheus for metric collection using the provided namespace")
+	cmd.PersistentFlags().StringVarP(&c.prometheusPort, "prometheus-port", "s", "9090",
```

Comment:
I'm a little concerned about `-n` and `-s` as the shortcode here. For one, I'm not sure why `-s` is for the port, and two I'm worried about these flags taking the short flags of more useful or common flags in the future. Should we not have shortcode on these for now?

Reply:
Good point. I've removed the short flags. We can do the same in the other telemetry PR too.